modules/understanding-upgrade-channels.adoc

```diff
@@ -82,6 +83,27 @@ and pre-production and production environments that participate in connected cus
 You can use the `stable-{product-version}` channel to upgrade from a previous minor version of
 {product-title}.
 
+[discrete]
+== eus-{product-version} channel
+
+In addition to the stable channel, certain minor versions of {product-title} offer an
+Extended Update Support[https://access.redhat.com/support/policy/updates/openshift#ocp4_phases] (EUS).
+These EUS versions extend the maintenance phase for customers with Premium Subscriptions
+to 14 months. {product-title} {product-version} is a minor version that includes EUS.
+
+Although there is no difference between stable-{product-version} and eus-{product-version}
+channels until {product-version} transitions to EUS phase, you can switch
+to the EUS channel as soon as it becomes available. When {product-title} {product-version}
+transitions to the EUS phase of its lifecycle, the stable-{product-version} channel will
+no longer receive updates. After you upgrade a to a version that is exclusive
+to the EUS channel, that cluster is no longer be eligible for minor version upgrades
+until upgrades to the next EUS version become available. Currently, upgrades between
+EUS versions requires a serial set of version upgrades, such as from 4.6 to 4.7 to 4.8 to 4.9 to 4.10.
```

Comment:
Is this an important aspect of the EUS channel?  Can we say something like:

> For at least the final six months (or whatever) of the EUS phase, an update pathway to the next EUS version will be available.

without going into detail about what we expect that update pathway will look like?

Reply:
I feel like we owe them some level of expectation setting, right now it's very likely this will involve four serialized upgrades. Without work getting put into planning to improve that I'd rather be upfront about the worst case.